Userland/Libraries/LibWeb/CSS/CSSFontFaceRule.cpp

```diff
@@ -31,8 +31,77 @@ CSSStyleDeclaration* CSSFontFaceRule::style()
 // https://www.w3.org/TR/cssom/#ref-for-cssfontfacerule
 String CSSFontFaceRule::serialized() const
 {
-    // FIXME: Implement this!
-    return "@font-face { /* FIXME: Implement CSSFontFaceRule::serialized()! */ }";
+    StringBuilder builder;
+    // The result of concatenating the following:
+
+    // 1. The string "@font-face {", followed by a single SPACE (U+0020).
+    builder.append("@font-face { "sv);
+
+    // 2. The string "font-family:", followed by a single SPACE (U+0020).
+    builder.append("font-family: "sv);
+
+    // 3. The result of performing serialize a string on the rule’s font family name.
+    builder.append(m_font_face.font_family());
+
+    // 4. The string ";", i.e., SEMICOLON (U+003B).
+    builder.append(";"sv);
```

Comment:
You can append the char directly.
```suggestion
    builder.append(';');
```